processor/attributekeyprocessor/factory.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attributekeyprocessor
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-service/config/configerror"
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
+	"github.com/open-telemetry/opentelemetry-service/consumer"
+	"github.com/open-telemetry/opentelemetry-service/processor"
+)
+
+var _ = processor.RegisterFactory(&factory{})
+
+const (
+	// The value of "type" Attribute Key in configuration.
+	typeStr = "attribute-key"
```

Comment:
Let go with simpler "attributes".

Reply:
"attributes" is already reserved by another processor, any other suggestions, please?

Reply:
You are right, it is used by addattributesprocessor.

@pjanotti any thoughts on naming? We need to clearly distinguish addattributesprocessor and attributekeyprocessor, yet keep the names short.

We are still open to changing the name for addattributesprocessor.

Reply:
`add-attributes` for the  first one and `attribute-key` would be my initial suggestions. It makes clear the purpose of each processor and makes easy to guess each package.

Reply:
Sounds good. @MohamedElqdusy let's go with @pjanotti 's suggestions. Please rename `addattributesprocessor` to use `add-attributes` as the typeStr.